index.html

```diff
@@ -645,6 +645,18 @@ <h4>OAuth 2.0</h4>
       </section>
 
     </section>
+    <section>
+      <h3>Options</h3>
+      <p>
+Some of the endpoints defined in the following sections accept an `options` object.
+All properties of the `options` object are OPTIONAL when configuring each endpoint,
+as these properties are intended to meet per-deployment needs that might vary.
+Thus, any given endpoint configuration MAY prohibit client use of some `options`
+properties in order to prevent clients from passing certain data to that endpoint.
+Likewise, an endpoint configuration MAY require that clients include some `options`
+properties.
```

Comment:
```suggestion
Some of the endpoints defined in the following sections accept an `options` object.
All properties of the `options` object are OPTIONAL when configuring each instance,
as these properties are intended to meet per-deployment needs that might vary.
Thus, any given instance configuration MAY prohibit client use of some `options`
properties in order to prevent clients from passing certain data to that instance.
Likewise, an instance configuration MAY require that clients include some `options`
properties.
```